ES/ESmy004.xml

```diff
@@ -631,6 +631,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                               <item xml:id="q17" n="17">
                                  <dim unit="leaf">7</dim>
                                  <locus from="123r" to="129v"/>
+                                 s.l.: 7, no stub
```

Comment:
if you do not specify which one is sl then the app cannot work for the diagrams and formulas

Reply:
But it is specified, s.l.: 7, no stub. No?

Reply:
sorry yes you are right it is the `<note>` that. at least in the preview, causes problems - it works if you move the note above the locus line